nautobot/extras/models/statuses.py

```diff
@@ -104,10 +101,12 @@ def _get_FIELD_color(self, field):
 
 class StatusModel(models.Model):
```

Comment:
Can we throw a deprecation warning decorator here?

Reply:
I can see if `class_deprecated_in_favor_of` can be used here as-is.